Can't install on Rust 1.59.0 stable

Running `cargo install --git https://github.com/LedgerHQ/cargo-ledger` with Rust stable `1.59.0` on Ubuntu 20.04 produces :

```console
error[E0432]: unresolved import `clap::Clap`
 --> src/main.rs:2:5
  |
2 | use clap::Clap;
  |     ^^^^^^^^^^ no `Clap` in the root

error: cannot determine resolution for the derive macro `Clap`
  --> src/main.rs:65:10
   |
65 | #[derive(Clap)]
   |          ^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error: cannot find attribute `clap` in this scope
  --> src/main.rs:66:3
   |
66 | #[clap(name = "Ledger NanoS load commands")]
   |   ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> src/main.rs:67:3
   |
67 | #[clap(version = "0.0")]
   |   ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> src/main.rs:68:3
   |
68 | #[clap(about = "Builds the project and emits a JSON manifest for ledgerctl.")]
   |   ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> src/main.rs:70:7
   |
70 |     #[clap(long)]
   |       ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> src/main.rs:71:7
   |
71 |     #[clap(value_name = "prebuilt ELF exe")]
   |       ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> src/main.rs:74:7
   |
74 |     #[clap(long)]
   |       ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> src/main.rs:75:7
   |
75 |     #[clap(about = concat!(
   |       ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> src/main.rs:82:7
   |
82 |     #[clap(subcommand)]
   |       ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot determine resolution for the derive macro `Clap`
  --> src/main.rs:86:10
   |
86 | #[derive(Clap)]
   |          ^^^^
   |
   = note: import resolution is stuck, try simplifying macro imports

error[E0599]: no function or associated item named `parse` found for struct `Cli` in the current scope
  --> src/main.rs:93:25
   |
69 | struct Cli {
   | ---------- function or associated item `parse` not found for this
...
93 |     let cli: Cli = Cli::parse();
   |                         ^^^^^ function or associated item not found in `Cli`
   |
   = help: items from traits can only be used if the trait is implemented and in scope
   = note: the following trait defines an item `parse`, perhaps you need to implement it:
           candidate #1: `Parser`

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: failed to compile `cargo-ledger v1.0.0 (https://github.com/LedgerHQ/cargo-ledger#e2894388)`, intermediate artifacts can be found at `/tmp/cargo-installAOQGU0`

Caused by:
  could not compile `cargo-ledger` due to 12 previous errors
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 
 [dependencies]
 cargo_metadata = "0.11.0"
-clap = "3.0.0-beta.1"
+clap = { version = "3.0.14", features = ["derive"] }
 goblin = "0.2.3"
 serde = "1.0"
 serde_derive = "1.0"

--- a/src/main.rs
+++ b/src/main.rs
@@ -1,5 +1,5 @@
 use cargo_metadata::Message;
-use clap::Clap;
+use clap::Parser;
 use std::process::{Command, Stdio};
 
 use std::env;
@@ -62,7 +62,7 @@ struct NanosMetadata {
     name: Option<String>,
 }
 
-#[derive(Clap)]
+#[derive(Parser)]
 #[clap(name = "Ledger NanoS load commands")]
 #[clap(version = "0.0")]
 #[clap(about = "Builds the project and emits a JSON manifest for ledgerctl.")]
@@ -72,7 +72,7 @@ struct Cli {
     use_prebuilt: Option<std::path::PathBuf>,
 
     #[clap(long)]
-    #[clap(about = concat!(
+    #[clap(help = concat!(
         "Should the app.hex be placed next to the app.json, or next to the input exe?",
         " ",
         "Typically used with --use-prebuilt when the input exe is in a read-only location.",
@@ -83,7 +83,7 @@ struct Cli {
     command: Option<SubCommand>,
 }
 
-#[derive(Clap)]
+#[derive(Parser)]
 enum SubCommand {
     /// Load the app onto a nano
     Load,

